app/views/summary/UKSitesSummary.scala

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
There are no tests to cover your first two and the last scenarios for this view. Please add those.

![image](https://github.com/hmrc/soft-drinks-industry-levy-registration-frontend/assets/5639505/7034d23d-bcfa-410f-ab21-494e33aa68a9)

Reply:
![image](https://github.com/hmrc/soft-drinks-industry-levy-registration-frontend/assets/75027964/530d3185-6987-46f2-8306-4f87854c08aa)